cfme/tests/containers/test_smart_management.py

```diff
@@ -74,29 +82,39 @@ def wait_for_tag(obj_inst):
                                forced_streams=['5.7'],
                                unblock=lambda test_item: test_item.obj != Container)])
 def test_smart_management_add_tag(provider, test_item):
-
+    logger.info("Setting smart mgmt tag to {obj_type}".format(obj_type=test_item.obj.__name__))
     # validate no tag set to project
     if test_item.obj is ContainersProvider:
         obj_inst = provider
     else:
         obj_inst = test_item.obj.get_random_instances(provider, count=1).pop()
 
+    logger.info("Selected object is \"{obj_name}\"".format(obj_name=obj_inst.name))
+
     regex = r"([\w\s|\-|\*]+:([\w\s|\-|\*])+)|(No.*assigned)"
     try:
         # Remove all previous configured tags for given object
+        logger.info("Starting cleaning old tags from "
+                 "object \"{obj_name}\"".format(obj_name=obj_inst.name))
```

Comment:
This indentation is not correct. Also you can use single quotes for the string so you don't need to escape the double quotes.

Reply:
@mkoura i fixed this